packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -1041,7 +1049,7 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
     const tx = await this.pkpHelperContract.write.mintNextAndAddAuthMethods(
       2, // key type
       [authMethod.authMethodType],
-      [authId],
+      [_authId],
```

Comment:
so do we not use the `accessToken` from the `authMethod`? looks like we just take the `authMethodType`